doc/eventhandler/customuserattributehandler.rst

```diff
@@ -0,0 +1,48 @@
+.. _customuserattributehandler:
+
+Custom User Attribute Module
+----------------------------
+
+.. index:: Custom User Attribute Handler, Handler Modules
+
+
+The custom user attribute handler module allows the adminitrator to set or delete
+custom user attributes automatically. A specific event can thus set or delete
+a custom user attribute without any further intervention of the administrator
+or helpdesk. These custom user attributes can even be used in policies or user resolvers.
+
+.. Note:: This event handler can also set attributes that can't be set or deleted manually
+          by the administrator or the helpdesk users.
+
+
+Possible Actions
+~~~~~~~~~~~~~~~~
+
+set_custom_user_attributes
+..........................
+
+This action sets a custom user attribute on a certain event.
+
+The action takes the options attrkey, attrvalue and user.
```

Comment:
Please mark the names of the options "attrkey" and "attrvalue", this way it is easier to read.

You can do this with the backticks "`" before and after the word.

Also check for this in the remaining text.